ballerina/LICENSE

```diff
@@ -17,171 +17,171 @@ ________________________________________________________________________________
 ballerina-command-1.3.15.jar                                                                        jar            apache2             
 commons-text-1.10.0.jar                                                                             bundle         apache2             
 compiler-0.8.9.jar                                                                                  jar            apache2             
-future-2201.7.3-20230926-134300-e31b2990.jar                                                        jar            apache2             
-ballerina-io-internal-2201.7.3-20230926-134300-e31b2990.jar                                         jar            apache2             
+future-2201.7.3.jar                                                        jar            apache2             
```

Comment:
shall we align the `jar` and `apache2` parts with other lines?

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-distribution/pull/4981/commits/dad61969bb83be2edb360671fd8dbd962e9fff92